tests/integration/test_integration_complete.py

```
@@ -0,0 +1,283 @@
+                # Show analysis excerpt
+                chatgpt_analysis = response_data.get('chatgpt_analysis', '{}')
+                try:
+                    # Handle both string and dict formats
+                    if isinstance(chatgpt_analysis, str):
+                        analysis = json.loads(chatgpt_analysis)
+                    elif isinstance(chatgpt_analysis, dict):
+                        analysis = chatgpt_analysis
+                    else:
+                        analysis = {}
+                    
+                    print(f"\n📋 Analysis Summary:")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)